_submissions/round-12/1/2015-02-26-simple-git-collaboration-existing-repo.md

```diff
@@ -0,0 +1,13 @@
+---
+date: 2015-02-26
+round: Round 12
+title: git collaboration from existing cloud repo
+author: Tessa Pierce 
+permalink: /2015/02/26/simple-git-collaboration-existing-repo
```

Comment:
Please use `/2015/02/simple-git-collaboration-existing-repo/` (year and month without day, and with a trailing slash) for the permalink.